src/backend/effects/builtin/chat.js

```diff
@@ -56,7 +72,15 @@ const effect = {
             messageId = trigger.metadata.eventData?.chatMessage?.id;
         }
 
-        await twitchChat.sendChatMessage(effect.message, effect.whisper, effect.chatter, !effect.whisper && effect.sendAsReply ? messageId : undefined);
+        if (effect.me) {
+            effect.message = `/me ${effect.message}`;
+        }
+
+        if (!effect.whisper && effect.whisperTarget != null) {
+            effect.whisperTarget = null;
+        }
+
+        await twitchChat.sendChatMessage(effect.message, effect.whisperTarget, effect.chatter, !effect.whisper && effect.sendAsReply ? messageId : undefined);
```

Comment:
with the above changes made, these lines can be reverted back to what it was before, which was just:
```js
await twitchChat.sendChatMessage(effect.message, effect.whisper, effect.chatter, !effect.whisper && effect.sendAsReply ? messageId : undefined);
```

Reply:
I've gone ahead and reverted this back to the original. Thank you again for your help!